backword-compatibility.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
Sorry, could you please move this file in the directory classes/Compatibility ? There are always some classes like that.